ui-library-filesharing-chat-composite/api/UploadFileToAzureBlobStore/index.ts

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { AzureFunction, Context, HttpRequest } from '@azure/functions';
+import * as multipart from 'parse-multipart';
+import HTTP_CODES from 'http-status-enum';
+
+const azureStorageConnectionString = process.env['azureStorageConnectionString'];
+let storageAccountName;
+try {
+  console.log("****"+azureStorageConnectionString);
+  storageAccountName = azureStorageConnectionString.match(/AccountName=([^;]+)/)[1];
+} catch {
+  console.warn('Azure storage connection string not provided');
```

Comment:
nit: This warning can be a bit misleading. How about `'Failed to parse Azure storage name from connection string'`

Also, I'd log this as `console.error` because the sample won't work properly without this.